doc/ch-run.rst

```diff
@@ -96,6 +96,11 @@ mounting SquashFS images with FUSE.
     bind-mounted into it. If this is specified, no such temporary files are
     created and the image’s files are exposed.
 
+  :code:`-q`, :code:`--quiet`
+    Be quieter; can be repeated. Incompatible with :code:`-v` and suppresses
+    :code:`--debug` regardless of option order. See the :ref:`FAQ entry on
```

Comment:
Does `ch-run` have `--debug`? I'm not seeing it.

Reply:
oops, good catch